README.md

```diff
@@ -42,19 +44,33 @@ iframe.src = vfrLink
 
 ### Iframe settings
 
-Add query parameters to the link to configure the optional settings:
+Add query parameters to the link to configure the component behavior. Virtual Fitting Room and 3D Viewer parameters differ slightly.
```

Comment:
Here we need to change Virtual Fitting Room -> Virtual Try-On (see the comment above)